.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2021: true,
+		node: true,
+	},
+	extends: [
+		'eslint:recommended',
+		'plugin:react/recommended',
```

Comment:
Do we need react here?

Reply:
No, it's not required, it's not a React project, I removed it in a new commit.